Jenkinsfile_CNP

```diff
@@ -35,6 +35,13 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
+// Vars for Kubernetes
+env.IDAM_API_BASE_URI = 'https://preprod-idamapi.reform.hmcts.net:3511'
```

Comment:
Now that these vars are in here, can we remove them from output.tf?

Reply:
depends, will we ever not need them in here, but need them in output.tf?
does output.tf export anywhere other than to Jenkins?

Reply:
I'm not sure. I read a confluence page a while back (wish I could find it again) that suggested it does output to other things but I've no idea what they would be. Maybe check with devops team

Reply:
I think it outputs to other apps if they refer to us in their infrastructure files, but that doesn't sound reasonable now that Im typing it